### ISSUE TITLE
Allow retrieval of actual number of dislikes from your database

### ISSUE THREAD

**Issue by HansBrende** [Anarios/return-youtube-dislike#755]
### Extension or Userscript?

Both

### Request or suggest a new feature!

There have been a bunch of news stories within the last day circulating on facebook along the lines of "The Little Mermaid Trailer has 1.5M downvotes because of racism!" 
E.g.: 
https://www.unilad.com/film-and-tv/little-mermaid-disney-live-action-youtube-trailer-20220913
https://www.dailymail.co.uk/femail/article-11208777/The-Little-Mermaid-live-action-trailer-hits-1-5-million-DISLIKES-just-two-days.html

Presumably they are using this plugin to get this number. However, one thing they all have in common is that they fail to realize that number is EXTRAPOLATED. Some of these stories incorrectly assert that youtube disabled the downvotes in response to the backlash, when in fact that is absurd, and they are not realizing a plugin was involved from the beginning.

So, either the video actually has 1.5M downvotes, or it could also be the case that someone found a way to artificially inflate the number of downvotes in your database, which artificially magnified the number of extrapolated votes to a huge degree. 

Say, for example, your database recorded 10 upvotes and then one single person came in and generated 100 downvotes somehow, then the 1.5M displayed number of downvotes on that youtube video would be 100X too high and the real number could be as low as 15K, in that example.

For this reason, it is very important to give access to the actual number of upvotes and downvotes recorded by your database.

### Ways to implement this!

Adding an endpoint to retrieve the *actual* number of downvotes and upvotes recorded by your database, or adding that information to an existing endpoint.

Presenting that information in the UI would be nice as well of course.

### Can you work on this?

- [ ] Yes
- [X] No

### Will you be available for follow-up questions to help developers implement this?

Yes

**Comment by theuserwhoisnotadeveloper** [comment]
downvotes? :) you came here from reddit ha? [here is](https://youtu.be/Nz9b0oJw69I?t=382) a video from LTT that shows how accurate the system is. as you can see it works pretty fine.

**Comment by tymondouglas** [comment]
Hi reddit

**Comment by HansBrende** [comment]
> downvotes? :) you came here from reddit ha? [here is](https://youtu.be/Nz9b0oJw69I?t=382) a video from LTT that shows how accurate the system is. as you can see it works pretty fine.

No, I did not come here from reddit; I have no clue what you're talking about. Your linked video shows nothing that we didn't already know. Irrelevant unless Walt Disney Studios volunteers the true dislike count: only they can see it.

**Comment by Tali64** [comment]
> 

Considering that other videos of the trailer show a similar like-to-dislike ratio, it's not that the extension is broken. However, this feature could be useful for some scenarios.
